distro/configuration/locations/locations-core_demo.csv

```diff
@@ -1,116 +1,60 @@
-f277520e-91af-4ea1-98a2-f011e6f85a37,,Ward 33,Ward 33,,Login Location,,,,,,,,,,,,,,,,
-84b2819f-cdb5-47f9-88f4-825131bb6afa,,Ward 34,Ward 34,,Login Location,,,,,,,,,,,,,,,,
-4738f9f4-ca34-4f9f-9e89-d7a2ba2b51a3,,Ward 35,Ward 35,,Login Location,,,,,,,,,,,,,,,,
-13b6681c-ae49-4f76-a5c6-24013d1bd7ec,,Ward 36,Ward 36,,Login Location,,,,,,,,,,,,,,,,
-3b6c2971-7a90-4076-baff-28f327e2ff9e,,Ward 37,Ward 37,,Login Location,,,,,,,,,,,,,,,,
-49d7321a-0ee6-419e-8c74-96554a1902e1,,Ward 38,Ward 38,,Login Location,,,,,,,,,,,,,,,,
-d5b5cf4e-e5fc-44c8-b19b-99bae46f7914,,Ward 39,Ward 39,,Login Location,,,,,,,,,,,,,,,,
-de7aa639-6cdb-4184-9171-235316d8aa6d,,Ward 40,Ward 40,,Login Location,,,,,,,,,,,,,,,,
-d212de33-7757-4899-b347-2cc693635b55,,Ward 41,Ward 41,,Login Location,,,,,,,,,,,,,,,,
-6534f987-9d1a-442a-a336-da6d62b25301,,Ward 42,Ward 42,,Login Location,,,,,,,,,,,,,,,,
-f8e92d51-9072-4cf0-9af2-7eb30b077c11,,Ward 43,Ward 43,,Login Location,,,,,,,,,,,,,,,,
-b120e051-c927-4ff6-9307-f90c8ded63ca,,Ward 44,Ward 44,,Login Location,,,,,,,,,,,,,,,,
-d1572bd9-ad63-4acc-9c2d-5e1dd0aeea99,,Ward 45,Ward 45,,Login Location,,,,,,,,,,,,,,,,
-0f9194e3-f7ff-438b-9319-387925cd6310,,Ward 46,Ward 46,,Login Location,,,,,,,,,,,,,,,,
-d14dd072-2f21-4340-b13d-845da00f8b28,,Ward 47,Ward 47,,Login Location,,,,,,,,,,,,,,,,
-3ebe38d4-ec5c-49c3-b498-0ca98c15f410,,Ward 48,Ward 48,,Login Location,,,,,,,,,,,,,,,,
-689d68a5-5857-48af-9f26-dc374fdf2997,,Ward 49,Ward 49,,Login Location,,,,,,,,,,,,,,,,
-736b08f9-94d6-4b50-ad58-6bc69b9cbfb8,,Ward 50,Ward 50,,Login Location,,,,,,,,,,,,,,,,
-4bc65b7a-bcd6-4829-860c-310aaf2c74db,,Ward 51,Ward 51,,Login Location,,,,,,,,,,,,,,,,
-14e1afa1-fa96-41b0-bfaa-a7aff2659ee6,,Ward 52,Ward 52,,Login Location,,,,,,,,,,,,,,,,
-8d243ead-936f-4bc8-b8a6-95baa2ed5217,,Ward 53,Ward 53,,Login Location,,,,,,,,,,,,,,,,
-9212bcda-32dc-4fb8-a517-8ade92908816,,Ward 54,Ward 54,,Login Location,,,,,,,,,,,,,,,,
-b8457ebe-48cd-4e6b-8139-23b269017d45,,Ward 55,Ward 55,,Login Location,,,,,,,,,,,,,,,,
-423ca603-a696-4a25-a00f-2e998df01a9f,,Ward 56,Ward 56,,Login Location,,,,,,,,,,,,,,,,
-9b35114a-a1cc-400c-ba1e-ee61538feeb7,,Ward 57,Ward 57,,Login Location,,,,,,,,,,,,,,,,
-aa661b2b-bf9f-4335-965c-9dcb5b6adf94,,Ward 58,Ward 58,,Login Location,,,,,,,,,,,,,,,,
-8874c0eb-be2b-4c9c-926c-bdd1e4394b99,,Ward 59,Ward 59,,Login Location,,,,,,,,,,,,,,,,
-2faca5a9-bfce-4cdc-8e09-690411230bcc,,Ward 60,Ward 60,,Login Location,,,,,,,,,,,,,,,,
\ No newline at end of file
+Uuid,Void/Retire,Name,Description,Parent,Tag|Login Location,Tag|Visit Location,Tag|Facility Location,Tag|Queue Location,Tag|Admission Location,Tag|Transfer Location,Attribute|9eca4f4e-707f-4bb8-8289-2f9b6e93803c,Attribute|Last Audit Date,Address 1,Address 2,Address 3,Address 4,Address 5,Address 6,City/Village,County/District,State/Province,Postal Code,Country,_order:1000
+8d6c993e-c2cc-11de-8d13-0010c6dffd0f,,Unknown Location,Unknown Location,,,,,,,,,,,,,,,,,,,,,
```

Comment:
The most noteworthy thing about the changes to locations is that it sets up a location hierarchy at "Ubuntu Hospital""

![image](https://github.com/user-attachments/assets/9cdf2771-74ff-4278-ae90-906c5c97709a)

This hierarchy is necessary for the Ward module to work, and modelling general way EMR-API "views the world": when you have multiple locations within a single facility the sites should be arranged as children of a main site location, and the main location (in this case "Ubuntu Hospital" should be tagged as a visit location.  In order for the Inpatient functionality to work and to allow admission and transfer between different locations, a patient's active visit needs to be at the parent site.

Reply:
(Note that I'm not sure what "Facility Location" means within this demo, so I left it as-is.